flaky-test: TestProcesses

I think short-lived processes can cause `TestProcesses` to fail intermittently.

```
    system_test.go:304: failed to get process info for PID=6387: open /proc/6387/stat: no such file or directory
--- FAIL: TestProcesses (0.01s)
```

Observed at https://github.com/elastic/go-sysinfo/pull/175.

--- a/system_test.go
+++ b/system_test.go
@@ -298,6 +298,7 @@ func TestProcesses(t *testing.T) {
 		case errors.Is(err, syscall.ESRCH),
 			errors.Is(err, syscall.EPERM),
 			errors.Is(err, syscall.EINVAL),
+			errors.Is(err, syscall.ENOENT),
 			errors.Is(err, fs.ErrPermission):
 			continue
 		case err != nil:

